RiotSwiftUI/Modules/Room/LocationSharing/View/LocationSharingView.swift

```diff
@@ -112,7 +122,7 @@ struct LocationSharingView: View {
                 // Hide for now until live location sharing is finished
                 if context.viewState.isLiveLocationSharingEnabled {
                     LocationSharingOptionButton(text: VectorL10n.locationSharingLiveShareTitle) {
-                        context.send(viewAction: .shareLiveLocation)
+                        showingTimerSelector = true
```

Comment:
I think you can keep a view action and put `var showingTimerSelector = false` in the `LocationSharingViewState`